src/main/java/uk/gov/pay/adminusers/persistence/entity/InviteTypeConverter.java

```diff
@@ -10,12 +10,12 @@ public class InviteTypeConverter implements AttributeConverter<InviteType, Strin
 
     @Override
     public String convertToDatabaseColumn(InviteType inviteType) {
-        return inviteType.getType();
+        return inviteType.name().toLowerCase();
```

Comment:
Probably from my lack of Java devving these days but why is it to lower case here?

Reply:
What's JSP? Surely not Java/Jakarta Server Pages 😂

Reply:
Nope, it's me using the wrong acronym when I mean JPA. 😆  I'll edit the various comments where I've gotten that wrong...